draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -546,17 +579,17 @@ a given host, the Path attribute cannot be relied upon for security (see
 #### The Secure Attribute {#attribute-secure}
 
 The Secure attribute limits the scope of the cookie to "secure" channels
-(where "secure" is outside the scope of this document). E.g., when a cookie has the Secure
-attribute, the user agent will include the cookie in an HTTP request only if
-the request is transmitted over a secure channel (typically HTTP over Transport
-Layer Security (TLS) {{RFC2818}}).
+(where "secure" is outside the scope of this document). for example, when a
+cookie has the Secure attribute, the user agent will include the cookie in an
+HTTP request only if the request is transmitted over a secure channel (typically
+HTTP over Transport Layer Security (TLS) {{RFC2818}}).
```

Comment:
This change seems unrelated?

Reply:
Huh - an accidental change. My bad - the cookie deletion section is the only important bit.